spec/dummy/app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,2 @@
+class StaticPagesController < ShiftCommerce::StaticPagesController
```

Comment:
We need to define a controller in the parent application to match those in the gem. These have to inherit from the gem's controller, but don't need anything else to be set.